common/dal/src/accessors/local.rs

```diff
@@ -21,6 +21,9 @@ use std::path::PathBuf;
 
 use async_compat::CompatExt;
 use common_base::tokio;
+use common_base::tokio::sync::RwLock;
+use common_cache::Cache;
+use common_cache::LruCache;
```

Comment:
`LruCache` is a in-memory cache, `LruDiskCache` is the disk cache.

Reply:
yes, we could have cache policies to decide which kind of data should be cached in mem or disk and when to do it.

Reply:
> yes, we could have cache policies to decide which kind of data should be cached in mem or disk and when to do it.

This means that a module for cache management is required.